atom/renderer/lib/web-view/web-view.coffee

```diff
@@ -236,6 +236,7 @@ registerWebViewElement = ->
   methods = [
     "getUrl"
     "getTitle"
+    "getFavicon"
```

Comment:
The `NativeImage` instance passed to WebView is an remote object, and would surprise users on some circumstances. We should probably just return a data url in this method.